schema.sql

```diff
@@ -3,6 +3,7 @@
 CREATE TABLE animals (
     id serial PRIMARY KEY,
     name varchar(100),
+    spicies varchar(100),
```

Comment:
- [ ] it is not good practice to change an existing table instead you're supposed to use the alter table query.